src/uct/ib/ud/base/ud_def.h

```diff
@@ -58,6 +58,7 @@ enum {
     UCT_UD_PACKET_FLAG_NACK    = UCS_BIT(27),
     UCT_UD_PACKET_FLAG_PUT     = UCS_BIT(28),
     UCT_UD_PACKET_FLAG_CTL     = UCS_BIT(29),
+    UCT_UD_PACKET_FLAG_FIN     = UCS_BIT(30),
```

Comment:
maybe send FIN packet by "control packet extended header", to not consume a reserved bit from the header? i.e add UCT_UD_PACKET_FIN and use uct_ud_ctl_hdr_t.
otherwise, pls update the ascii art below to represent the new FIN packet

Reply:
done